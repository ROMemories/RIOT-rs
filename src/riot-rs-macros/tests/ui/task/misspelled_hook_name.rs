#![no_main]
#![feature(impl_trait_in_assoc_type)]
#![feature(type_alias_impl_trait)]
#![feature(used_with_arg)]

// As the macro will fail, this import will not get used
#[allow(unused_imports)]
use riot_rs::usb::UsbBuilderHook;

// FAIL: misspelled hook name
#[riot_rs::task(autostart, usb_builder_hooook)]
async fn main() {}
