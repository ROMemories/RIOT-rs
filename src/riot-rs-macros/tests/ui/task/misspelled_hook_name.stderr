error: unsupported parameter (`autostart`, `peripherals`, `pool_size`, and hooks `usb_builder_hook` are supported)
  --> tests/ui/task/misspelled_hook_name.rs:10:28
   |
10 | #[riot_rs::task(autostart, usb_builder_hooook)]
   |                            ^^^^^^^^^^^^^^^^^^
